packages/oruga-next/src/components/autocomplete/Autocomplete.vue

```diff
@@ -156,6 +156,13 @@ export default defineComponent({
         keepOpen: Boolean,
         /** Add a button/icon to clear the inputed text */
         clearable: Boolean,
+        /**
+         * Icon name to be added on the clear button
+         */
+        clearIcon: {
+            type: String,
+            default: 'close-circle'
```

Comment:
Can you use the syntax that read it from global config in order to change it in the initial config?